### PR TITLE
Update DG appendix and personal portfolio page

### PR DESCRIPTION
**Pull request by AhQuanz**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -735,6 +735,83 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+1. Deleting a customer in CHIM
+    1. Prerequisites: There are some customer in CHIM, customer with phone number provided exist in CHIM.
+    2. Test case: `deletecustomer p/87438807`
+        Expected: The customer with phone number of 87438807 is deleted.
+            If displayed list is not customer list , use command `listcustomer` to verify. Details of the deleted customer is as shown in the status message.
+    3. Test case: `deletecustomer`
+        Expected: No customer is deleted, Error messsage with command requirements is shown in the status message.
+    4. Other incorrect `deletecustomer` to try: `deletecustomer n\x`(where x is a phone number does not exist in CHIM)
+
+### Finding customers
```
Comment (daniellau88) [inline docs/DeveloperGuide.md:832]: Do we include these? I think would be best if we just leave it out

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -735,6 +735,83 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+testers are expected to do more *exploratory* testing.
 
+</div>
+
+### Launch and shutdown
+1. Initial launch
+    1. Download the latest release jar file from [here](https://github.com/AY2021S2-CS2103-W16-2/tp/releases).
+    2. Copy the downloaded jar file into an empty folder.
+    3. Nagviate your command prompt to the folder containing the downloaded jar file and run the command `java -jar <fileName>`
+       . Lunach the application by double-clicking only as a last resort if the command does not work.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: Typos for navigate and launch

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -735,6 +735,83 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+### Deleting a customer
+1. Deleting a customer in CHIM
+    1. Prerequisites: There are some customer in CHIM, customer with phone number provided exist in CHIM.
+    2. Test case: `deletecustomer p/87438807`
+        Expected: The customer with phone number of 87438807 is deleted.
+            If displayed list is not customer list , use command `listcustomer` to verify. Details of the deleted customer is as shown in the status message.
+    3. Test case: `deletecustomer`
+        Expected: No customer is deleted, Error messsage with command requirements is shown in the status message.
+    4. Other incorrect `deletecustomer` to try: `deletecustomer n\x`(where x is a phone number does not exist in CHIM)
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: should be `deletecustomer p/x` instead of `n\x`

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -735,6 +735,83 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    1. launch the application and run either `listorders` , `listcustomers` or `listcheeses`. Close the window.
+    2. Re-launch the application by following either the command or by double-clicking.
+        Expected: The most recent list depending on last `list` command will be displayed.
+
+### Adding a customer
+1. Adding a customer into CHIM
+    1. Prerequisites: Arguments are valid and compulsory parameters are provided. No duplicated phone number exist in CHIM.
+    2. Test case: `addcustomer n/Tim p/96284715 e/tim@example.com a/Blk 55 Woodlands Street 99, #99-23`
+        Expected: item is added into CHIM. If displayed list is customer list , the new customer appears at the bottom.
+            Otherwise, run command `listcustomers`. Details of the new customer is shown in the status message.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: Maybe can rephrase since adding a customer always displays the customer list? Similar for deleting as well

**Review by laurenlhy — DISMISSED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (AhQuanz) [reply docs/DeveloperGuide.md:832]: based on the sample and the team`s UG i got for PE dry run , they had this section.

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (AhQuanz) [reply docs/DeveloperGuide.md]: sure. Will update in next commit

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (AhQuanz) [reply docs/DeveloperGuide.md]: Will update in next commit

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (AhQuanz) [reply docs/DeveloperGuide.md]: Will update in next commit

File: docs/team/ahquanz.md [hunk 5]
```diff
@@ -5,4 +5,45 @@ title: Li Quan's Project Portfolio Page
...
+
+* **New Feature**: Design and implementation of the initial model classes to be used for the project in v1.1.
+    * What it does: Provides models for team members to use during their implementation of the different features.
+    * Justification: This feature helps in the future development process since basic classes are provided and 
+      the architecture of the project has already been discussed and decided.
+    * Highlights: The designing of the initial model classes was slightly challenging because we are starting from scratch, 
+      team members had different ideas of how the model classes should be designed and implemented. 
+      
+* **New Feature**: Marking an order as complete in the CHIM app.
+    * What id does: It allows the user of the CHIM app to mark an order as complete by a command.
```
Comment (daniellau88) [inline docs/team/ahquanz.md]: What it does

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+</div>
 
 ### Launch and shutdown
+1. Initial launch
+    1. Download the latest release jar file from [here](https://github.com/AY2021S2-CS2103-W16-2/tp/releases).
+    2. Copy the downloaded jar file into an empty folder.
+    3. Navigate your command prompt to the folder containing the downloaded jar file and run the command `java -jar <fileName>`.
+
+        Lunch the application by double-clicking only as a last resort if the command does not work.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: Launch typo

File: docs/team/ahquanz.md [hunk 7]
```diff
@@ -5,4 +5,46 @@ title: Li Quan's Project Portfolio Page
 
 ## Project: Chesse Inventory Management (CHIM)
```
Comment (laurenlhy) [inline docs/team/ahquanz.md]: Cheese typo

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+       Expected: all orders in CHIM are listed.
+
+### Add a Cheese
+1. Adding a cheese into CHIM.
+    1. Prerequisites:
+        - Arguments are valid and compulsory parameters are provided.
+        - If manufacture_date is given , it must be any date up to current date , not in the future. 
+        - If manufacture_date is not given, it defaults to current date.
+        - expiry_date if given must occur after manufacture_date.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: Should use `MANUFACTURE_DATE` and `EXPIRY_DATE` instead

**Review by laurenlhy — COMMENTED** [review]
Some minor typos

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+<div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
+testers are expected to do more *exploratory* testing.
+
+</div>
 
 ### Launch and shutdown
+1. Initial launch
+    1. Download the latest release jar file from [here](https://github.com/AY2021S2-CS2103-W16-2/tp/releases).
+    2. Copy the downloaded jar file into an empty folder.
+    3. Navigate your command prompt to the folder containing the downloaded jar file and run the command `java -jar <fileName>`.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: change "fileName" to chim.jar?

File: docs/DeveloperGuide.md [hunk 10]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+        Launch the application by double-clicking only as a last resort if the command does not work.
+
+2. Saving window size and location preferences.
+    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+    2. Re-launch the application by following either the command or by double-clicking.
+
+        Expected: The most recent window size and location is retained.
+
+3. Saving application launch display list.
+    1. launch the application and run either `listorders` , `listcustomers` or `listcheeses`. Close the window.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: Change to capital L for "launch"

File: docs/DeveloperGuide.md [hunk 11]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    2. Test case: `deletecustomer p/87438807`.
+
+        Expected: The customer with phone number of 87438807 is deleted. Details of the deleted customer is as shown in the status message.
+
+    3. Test case: `deletecustomer`.
+
+        Expected: No customer is deleted, Error messsage with command requirements is shown in the status message.
+
+    4. Other incorrect `deletecustomer` to try: `deletecustomer p\x`(where x is a phone number does not exist in CHIM).
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: forward slash instead of backslash in `deletecustomer p\x`

File: docs/DeveloperGuide.md [hunk 12]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+        Expected: Error message with command requirements is shown in the status message. 
+
+    5. Other incorrect `findcustomer` to try: include any invalid keyword prefix like `findcustomer n/A B C u/TEST`.
+
+        Expected: similar to previous.
+
+### Listing Customers
+1. Listing all customers in CHIM.
+    1. Test case: `listcustomer`.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: testcase should be `listcustomers`

File: docs/DeveloperGuide.md [hunk 13]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    3. Test case: `editorder 1 q/10 d/9/4/2021`.
+
+        Expected: The quantity of the first order in `listorders` will be edited to `10` and order date will be updated to `9/4/2021`.
+
+    4. Test case: `editorder 0`.
+
+        Expected: No order is edited, Error message with command requires is shown in the status message.
+
+    5. Test case: `editorder 1 d\31/12/9999`.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: forward slash instead of backslash? `d/31/12/9999`

File: docs/DeveloperGuide.md [hunk 14]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+       Expected: all orders in CHIM are listed.
+
+### Add a Cheese
+1. Adding a cheese into CHIM.
+    1. Prerequisites:
+        - Arguments are valid and compulsory parameters are provided.
+        - If `MANUFACTURE_DATE` is given , it must be any date up to current date , not in the future. 
+        - If `MANUFACTURE_DATE` is not given, it defaults to current date.
+        - `EXPIRY_DATE` if given must occur after manufacture_date.
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: manufacture_date should be `MANUFACTURE_DATE`

File: docs/DeveloperGuide.md [hunk 15]
```diff
@@ -735,6 +735,335 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+
+    3. Test case: `editcheese y t/Parmesan` (where y is the index of a cheese that is assigned)
+
+        Expected: No cheese is edited , Error message will be displayed in the status message
+
+    4. Test case: `editcheese x d/31/12/9999` (where x is the index of the cheese that is not assigned)
+
+        Expected: similar to previous 
+    
+    5. Test case: `editcheese x e/DATE` (where DATE occurs before manufacture_date)
```
Comment (laurenlhy) [inline docs/DeveloperGuide.md]: manufacture_date should be `MANUFACTURE_DATE`

**Review by laurenlhy — COMMENTED** [review]
A few more typos to fix

**Review by daniellau88 — APPROVED** [review]
LGTM

**Pull request merged**